src/index.d.ts

```diff
@@ -393,197 +415,852 @@ declare namespace admin.app {
+
+    /**
+     * Whether or not the user's primary email is verified.
+     */    
     emailVerified?: boolean;
+
+    /**
+     * The user's unhashed password.
+     */
     password?: string;
+
+    /**
+     * The user's primary phone number. 
+     */
     phoneNumber?: string | null;
+
+    /**
+     * The user's photo URL. 
+     */
     photoURL?: string | null;
   }
 
+  /**
+   * Interface representing the properties to set on a new user record to be
+   * created.
+   */ 
   interface CreateRequest extends UpdateRequest {
+    
+    /**
+     * The user's `uid`.
+     */
     uid?: string;
   }
 
+  /**
+   * Interface representing a decoded Firebase ID token, returned from the
+   * {@link https://firebase.google.com/docs/reference/admin/node/admin.auth.Auth#verifyIdToken `verifyIdToken()`} method.
+   *
+   * Firebase ID tokens are OpenID Connect spec-compliant JSON Web Tokens (JWTs).
+   * See the
+   * [ID Token section of the OpenID Connect spec](http://openid.net/specs/openid-connect-core-1_0.html#IDToken)
+   * for more information about the specific properties below.
+   */ 
   interface DecodedIdToken {
+
+    /**
+     * The audience for which this token is intended.
+     *
+     * This value is a string equal to your Firebase project ID, the unique
+     * identifier for your Firebase project, which can be found in [your project's
+     * settings](https://console.firebase.google.com/project/_/settings/general/android:com.random.android).
+     */ 
     aud: string;
+
+    /**
+     * Time, in seconds since the Unix epoch, when the end-user authentication
+     * occurred.
+     *
+     * This value is not when this particular ID token was created, but when the
+     * user initially logged in to this session. In a single session, the Firebase
+     * SDKs will refresh a user's ID tokens every hour. Each ID token will have a
+     * different [`iat`](#iat) value, but the same `auth_time` value.
+     */
     auth_time: number;
+
+    /**
+     * The ID token's expiration time, in seconds since the Unix epoch. That is, the
+     * time at which this ID token expires and should no longer be considered valid.
+     *
+     * The Firebase SDKs transparently refresh ID tokens every hour, issuing a new
+     * ID token with up to a one hour expiration.
+     */
     exp: number;
+
+    /**
+     * Information about the sign in event, including which sign in provider was
+     * used and provider-specific identity details.
+     *
+     * This data is provided by the Firebase Authentication service and is a
+     * reserved claim in the ID token.
+     */
     firebase: {
+
+      /**
+       * Provider-specific identity details corresponding
+       * to the provider used to sign in the user.
+       */ 
       identities: {
         [key: string]: any;
       };
+
+      /**
+       * The ID of the provider used to sign in the
+       *   user. One of `"anonymous"`, `"password"`, `"facebook.com"`, `"github.com"`,
```

Comment:
This block does not seem to align with the above statement. You have 2 additional spaces.

Reply:
this is still not aligned.